test/org/sagebionetworks/bridge/play/controllers/SchedulePlanControllerMockTest.java

```diff
@@ -162,7 +163,7 @@ public void getSchedulePlan() throws Exception {
     
     @Test
     public void deleteSchedulePlan() throws Exception {
-        Result result = controller.deleteSchedulePlan("GGG");
+        Result result = controller.deleteSchedulePlan("GGG", "false");
```

Comment:
Missing unit test for logical delete vs physical delete. Did we forget to add a file?